generators/server/templates/quarkus/src/main/java/package/web/rest/vm/LoggerVM.java.ejs

```diff
@@ -0,0 +1,78 @@
+<%#
+ Copyright 2020-2021 the original author or authors from the JHipster project.
+
+ This file is part of the JHipster project, see https://www.jhipster.tech/
+ for more information.
+
+ Licensed under the Apache License, Version 2.0 (the "License");
+ you may not use this file except in compliance with the License.
+ You may obtain a copy of the License at
+
+      http://www.apache.org/licenses/LICENSE-2.0
+
+ Unless required by applicable law or agreed to in writing, software
+ distributed under the License is distributed on an "AS IS" BASIS,
+ WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ See the License for the specific language governing permissions and
+ limitations under the License.
+-%>
+package <%=packageName%>.web.rest.vm;
+
+import java.util.Optional;
+import java.util.logging.Level;
+import java.util.logging.Logger;
+
+public class LoggerVM {
+  private String name;
```

Comment:
Couldn't we use the "public" (or at least package) Quarkus idiom?

there is no added value to have "dummies" getter/setter in this context.